packages/ndla-ui/src/LicenseByline/EmbedByline.tsx

```diff
@@ -175,25 +129,32 @@ const EmbedByline = ({
   const authors = getLicenseCredits(copyright);
   const captionAuthors = Object.values(authors).find((i) => i.length > 0) ?? [];
 
+  const authorAndLicenseLink = () => {
+    return (
+      <>
+        {` ${t(`embed.type.${type}`)}${captionAuthors.length ? ": " : ""}`}
+        {captionAuthors.map((author) => author.name).join(", ")}
+        {license ? (
+          <>
+            {" / "}
+            <LicenseLink license={license} />
+          </>
+        ) : null}
+      </>
+    );
+  };
+
   return (
-    <BylineWrapper
-      data-top-rounded={topRounded}
-      data-hide-on-large-screens={hideOnLargeScreens}
-      data-bottom-rounded={bottomRounded}
```

Comment:
Hvis du fjerner disse er det sikkert lurt å fjerne prop-typene som tillater at dette sendes inn til komponenten også

Reply:
Og tilhørende css!

Reply:
De er fortsatt i bruk litt lenger opp i koden

Reply:
Gir det mening at byline kun skal være rounded dersom man ikke klarer å laste en embed?

Reply:
Den blir helt annerledes da, for det blir mer en "tilbakemelding" til brukeren at embed feilet. Hedvig sa at den skal bare bli sånn som den var
![image](https://github.com/NDLANO/frontend-packages/assets/110535103/2c65a485-8a6a-417f-88f4-1a8f91182413)